src/pybind/mgr/dashboard/frontend/src/app/core/navigation/notifications/notifications.component.scss

```diff
@@ -10,7 +10,7 @@
 
 a {
   .dot {
-    background-color: vv.$primary;
+    background-color: vv.$primary-light;
```

Comment:
I guess that most frameworks leaned towards a numeric suffix (`-300`) instead of a descriptive one (`-light`, `-dark`) is that it makes it harder to transition from light to dark themes (in a dark theme the lighter background color would become a dark one). However, with numbers you just need to reverse the meaning of the scale (with light themes 0 means darker and 1000 means lighter, while with dark themes it's the opposite).